simple_pdf_generator/src/lib.rs

```diff
@@ -114,13 +113,11 @@ struct ChromiumInstance {
 
 impl ChromiumInstance {
     async fn new() -> Self {
-        let options = BrowserConfig::builder();
-        let options = if NO_SANDBOX.load(Ordering::Relaxed) {
-            options.no_sandbox()
-        } else {
-            options
-        };
-        let options = options.build().expect("Invalid browser options.");
+        let options = 
```

Comment:
The `no_sandbox` flag is useful in containered contexts such as `LXC` and `Docker` (when the container is unprivileged but you have only the `root` user inside the container), it should therefore be left as it is

Reply:
altered it back to how it was